packages/firestore/src/api/database.ts

```diff
@@ -101,6 +101,7 @@ import {
 const DEFAULT_HOST = 'firestore.googleapis.com';
 const DEFAULT_SSL = true;
 const DEFAULT_TIMESTAMPS_IN_SNAPSHOTS = false;
+const DEFAULT_SYNCHRONIZE_TABS = false;
```

Comment:
Might be worth separating the .settings() default settings from the .enablePersistence() default settings...  Either via naming (DEFAULT_PERSISTENCE_SYNCHRONIZE_TABS) or just whitespace / comments:

```
// settings() defaults:
...

// enablePersistence() defaults:
...
```

Reply:
That makes sense. I added the comment strings.